rest/config.go

```diff
@@ -760,6 +785,15 @@ func ParseCommandLine(runMode SyncGatewayRunMode) {
 			config.SkipRunmodeValidation = *skipRunModeValidation
 		}
 
+		if defaultLogFilePathFlag != nil {
+			defaultLogFilePath = *defaultLogFilePathFlag
+		}
+
+		// Log HTTP Responses if verbose is enabled.
+		if verbose != nil && *verbose {
```

Comment:
This seems a bit unrelated to this PR.. was it supposed to sneak in?  Also, I wish we could just remove this option and let people specify `HTTP+` directly, but that's probably a different conversation.

Reply:
I think I noticed it was a regression in behavior for 2.1 as I was writing this PR. (i.e. --verbose was being ignored after the new logging went in)

I can split it out if necessary.

Reply:
Probably not needed, I was mainly making sure it was intended to be included.